infra/horreum-dev-services/deployment/src/main/java/io/hyperfoil/tools/horreum/dev/services/deployment/HorreumDevServicesProcessor.java

```diff
@@ -25,11 +25,10 @@
 import io.quarkus.deployment.builditem.*;
 import io.quarkus.deployment.console.ConsoleInstalledBuildItem;
 import io.quarkus.deployment.console.StartupLogCompressor;
-import io.quarkus.deployment.dev.devservices.GlobalDevServicesConfig;
 import io.quarkus.deployment.logging.LoggingSetupBuildItem;
 
 @BuildSteps(onlyIfNot = IsNormal.class, onlyIf = { HorreumDevServicesProcessor.IsEnabled.class,
-        GlobalDevServicesConfig.Enabled.class })
+        io.quarkus.deployment.dev.devservices.DevServicesConfig.Enabled.class })
```

Comment:
It's not ideal to use full package name imo, but we have a class named `io.hyperfoil.tools.horreum.dev.services.deployment.config.DevServicesConfig` imported already. wdyt @lampajr ?